src/flows/CostCalculator/hooks.ts

```diff
@@ -37,6 +37,7 @@ export const useCostCalculatorCountries = () => {
 };
 
 export const useCalculatorLoadRegionFieldsSchemaForm = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Doesn't matter for now, as this will be deleted [here](https://github.com/remoteoss/remote-flows/pull/19).